### DISCUSSION TITLE
About 5G NR module

### DISCUSSION THREAD

**Discussion by zhengxf99** [NVlabs/sionna#104] [Q&A] [answered]
Hi

Many thanks for the release of well-designed 5G NR module and related tutorials. They are really helpful.

May I ask is there any plan to support more 5G NR features as listed below in future releases of Sionna?

- PDSCH

- standard-compliant CSI feedback

- link adaptation

- etc.

Thanks a lot!

**Comment by jhoydis** [comment, marked as answer]
Hi,

These are all interesting features. However, we do not have concrete release plans for any of them yet.

**Reply by zhengxf99** [reply]
Thanks for the response
